modules/test/playwright/tests/calendar-web/miniCalendar.spec.ts

```diff
@@ -110,3 +114,69 @@ test('ensure that mini calendar accessibility properties are maintained after ch
 		'polite'
 	);
 });
+
+test('ensure that next and previous buttons acessibility label in the mini calendar are translated to portuguese', async ({
+	accountSettingsPage,
+	apiHelpers,
+	calendarWidgetPage,
+	page,
+	pageEditorPage,
+	site
+}) => {
+	
+	const company =
+		await apiHelpers.jsonWebServicesCompany.getCompanyByWebId(
+			'liferay.com'
+		);
+
+	const defaultUser = 
+		await apiHelpers.headlessAdminUser.getUserAccountByEmailAddress(
+			'test@liferay.com'
+		);
+
+	const user = await createUserWithPermissions({
+		apiHelpers,
+		rolePermissions: [
+			{
+				actionIds: ['VIEW'],
+				primaryKey: company.companyId,
+				resourceName: 'com.liferay.portal.kernel.model.Layout',
+				scope: 1,
+			},
+		],
+	});
+
+	await performUserSwitch(page, user.alternateName);
+
+	await page.goto(liferayConfig.environment.baseUrl);
+
+	await page.locator('button[data-qa-id="userPersonalMenu"]').click();
+
+	await page.getByRole('menuitem', {name: 'Account Settings'}).click();
+
+	await accountSettingsPage.selectAccountLanguage('pt_BR');
+
+	await pageEditorPage.goto(layout, site.friendlyUrlPath);
+	
+	await expect(
+		calendarWidgetPage.miniCalendarNextMonthButton
+	).toHaveAccessibleName('Ir para o próximo mês');
+
+	await expect(
+		calendarWidgetPage.miniCalendarPastMonthButton
+	).toHaveAccessibleName('Ir para o mês anterior');
+
+	await page.goto('en');
```

Comment:
is this the clean up? place the test body until the assertion inside a `try`, and the cleanup inside a `finally`, or you risk leaving a new user behind, which is usually non-critical, but should be avoided anyway. Thanks

Reply:
Yeah, it makes sense. Done!